src/ServiceFabric.Mocks/MissingActorEventArgs.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using Microsoft.ServiceFabric.Actors;
+
+namespace ServiceFabric.Mocks
+{
+          
+
+    public class MisingActorEventArgs : EventArgs
```

Comment:
I guess the only way to do it would be to assume there is only one concrete actor that implements a specific interface.  In some cases I have had more generic interfaces where an actor needs to call the original caller actor back but doesn't need the specific type.  If there is a worker actor or something where different types of actors may initialize it but the worker actor needs to report back to the original actor in a generic way with results.